iothub/service/tests/RetryPolicies/IotHubServiceRetryPolicyBaseTests.cs

```diff
@@ -45,7 +45,7 @@ public void RetryPolicyBase_ObservesInifiniteRetries()
         [TestMethod]
         [DataRow(true)]
         [DataRow(false)]
-        public void RetryPolicyBase_IotHubException_ReturnsTrueWhenTransient(bool isTransient)
+        public void RetryPolicyBase_IotHubServiceException_ReturnsTrueWhenTransient(bool isTransient)
```

Comment:
correcting the test method name as we don't have `IotHubException` context any more.